static/src/javascripts/projects/commercial/modules/glabs-traffic-driver-slot.js

```diff
@@ -26,7 +26,36 @@ const rules = {
     minAbove: 0,
     minBelow: 0,
     clearContentMeta: 0,
-    selectors: null,
+    selectors: {
+        ' .element-rich-link': {
+            minAbove: 200,
+            minBelow: 200,
```

Comment:
I'm never sure about this :/ Having had a look at [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas) it looks like it is fine (or encouraged(?)) in this case, but not when writing actual JSON.